apps/web-giddh/src/app/shared/subscription-upgrade-button/subscription-upgrade-button.component.ts

```diff
@@ -0,0 +1,84 @@
+import { ReplaySubject } from 'rxjs';
+import { Component, EventEmitter, Input, OnDestroy, Output } from '@angular/core';
+import { RestrictedModules } from '../../app.constant';
+import { Router } from '@angular/router';
+@Component({
+    selector: 'restricted-module-message',
+    styleUrls: [`./subscription-upgrade-button.component.scss`],
+    templateUrl: './subscription-upgrade-button.component.html'
+})
+export class SubscriptionUpgradeButtonComponent implements OnDestroy {
+    /** Active company details */
+    @Input() public activeCompany: any;
```

Comment:
fetch active company, common locale in this component only instead of taking input